app/views/cancel_declaration.scala.html

```diff
@@ -16,61 +16,90 @@
 
 @import controllers.routes._
 @import forms.CancelDeclaration
-@import uk.gov.hmrc.play.views.html._
-
-@import views.components.inputs.RadioOption
-@import forms.cancellation.CancellationChangeReason
 @import forms.cancellation.CancellationChangeReason.{NoLongerRequired, Duplication, OtherReason}
 @import forms.Choice
 @import forms.Choice.AllowedChoiceValues.CancelDec
 @import views.Title
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.BackButton
+@import views.html.components.gds._
 
-@this(main_template: views.html.main_template)
+@this(
+        govukLayout: gdsMainTemplate,
+        govukButton: GovukButton,
+        govukRadios: govukRadios,
+        errorSummary: errorSummary,
+        pageTitle: pageTitle,
+        exportsInputText: exportsInputText,
+        exportsInputTextArea: exportsInputTextArea,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(form: Form[CancelDeclaration])(implicit request: Request[_], messages: Messages)
 
-@main_template(
-    title = Title("cancellation.title")
+@govukLayout(
+    title = Title("cancellation.title"),
+    backButton = Some(BackButton(messages("site.backToSelectionPage"), controllers.routes.ChoiceController.displayPage(Some(Choice(CancelDec)))))
 ) {
 
-    @helper.form(action = CancelDeclarationController.onSubmit(), 'autoComplete -> "off") {
-        @helper.CSRF.formField
-
-        @components.back_link(controllers.routes.ChoiceController.displayPage(Some(Choice(CancelDec))))
+    @formHelper(action = CancelDeclarationController.onSubmit(), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
-        @components.error_summary(form.errors)
+        <div class="govuk-grid-column-two-thirds">
 
-        @components.heading(title = messages("cancellation.title"))
+            @pageTitle(messages("cancellation.title"))
 
-        @components.fields.field_text(
-            field = form("functionalReferenceId"),
-            label = Some(messages("cancellation.functionalReferenceId")),
-            labelClass = Some("bold-small")
-        )
+            @exportsInputText(
+                field = form("functionalReferenceId"),
+                labelKey = "cancellation.functionalReferenceId",
+                labelClasses = ""
```

Comment:
Should there not be a label class?  Prototype seems to be `govuk-label`

Reply:
The `govuk-label` class is added by default:
```
<label class="govuk-label" for="functionalReferenceId">
        Enter the DUCR
      </label>
```

Reply:
You are right.  My bad.  With my suggestions merged in we now have double label!
```<label class="govuk-label govuk-label" for="mrn">
        Enter the Movement Reference Number (MRN)
      </label>```

I think that error is repeated elsewhere on our pages.  Not a biggy.  Maybe we can clean up the exportsInputText component later.